Hyperbolic_surface_triangulation_2/demo/Hyperbolic_surface_triangulation_2_Demo/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.1...3.15)
+
+project( demo )
```

Comment:
make the name more specific. Also the one of the cpp

Reply:
done